react-ui/web/src/components/analysis/integrated_site/BackToMainButton.js

```diff
@@ -11,18 +11,164 @@ const goBack = history => e => {
+                        onClose={hourIntervalHandleClose} 
+                    >
+                        {
+                            hour_interval.map((row, index) => {
+                                const { label } = row;
+                                return (
+                                    <MenuItem
+                                        // eslint-disable-next-line react/no-array-index-key
+                                        key={`hour_interval_${index}`} 
+                                        onClick={() => hourIntervalSelected(row)}>
+                                        {label}
+                                    </MenuItem>
+                                );
+                            })
+                        }
+                
+                    </Menu>
+                </Fragment>
+            );
+        }
+
+        return ("");
+    };
 
     return (
-        isWidthDown(width, "sm") ? (<Button
-            variant="contained"
-            color="primary"
-            size="small" 
-            onClick={goBack(history)}
-        >
-            <ArrowBackIos style={{ fontSize: 16 }}/> Back to site list
-        </Button>) : (<IconButton
+        isWidthDown(width, "sm") ? (<Fragment>
+            <Button
+                variant="contained"
+                color="primary"
+                size="small" 
+                onClick={goBack(history)}
+                style={{ marginRight: 8 }}
+            >
+                <ArrowBackIos style={{ fontSize: 16 }}/> Back to site list
+            </Button>
+            <Tooltip title="Select date range" arrow>
+                <Button 
+                    variant="contained"
+                    color="primary"
+                    size="small" 
+                    aria-controls="simple-menu"
+                    aria-haspopup="true"
+                    onClick={dateRangeHandleClick}
+                    style={{ marginRight: 8 }}
+                >
+                    {selected}
+                </Button>
+            </Tooltip>
+            <Menu
+                id="simple-menu"
+                anchorEl={dateRangeAnchor}
+                keepMounted
+                open={Boolean(dateRangeAnchor)}
+                onClose={dateRangeHandleClose} 
+            >
+                {
+                    date_range_options.map((row, index) => {
+                        const { label } = row;
+                        return (
+                            <MenuItem
+                                // eslint-disable-next-line react/no-array-index-key
+                                key={`date_range_${index}`} 
+                                onClick={() => dateRangeSelected(row)}>
+                                {label}
+                            </MenuItem>
+                        );
+                    })
+                }
+                
+            </Menu>
+            {SubsurfaceHourInterval()}
```

Comment:
Add if chartType here
```suggestion
            { chartType === "subsurface" && SubsurfaceHourInterval() }
```